tests/unit/utils/unit_tests_string_utils.rb

```diff
@@ -185,6 +186,56 @@ def test_string_utils_str_to_bool_raise_argument_error_message
     assert_equal("invalid value for Boolean(): \"#{value}\"", exception.message)
   end
 
+  def test_string_utils_sha1_hash_bool_true_value
+    value = true
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_bool_false_value
+    value = false
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_int_value
+    value = 123
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_float_value
+    value = 3.14
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_range_value
+    value = (1..10)
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_array_value
+    value = [1, 2, 3, 4, 5]
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_hash_value
+    value = {:a => true, :b => false, :c => 1, :d => 2.0, :e => (0..9), :f => [0, 1, 2]}
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_symbol_value
+    value = :my_symbol
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_string_value
+    value = "unit_test_text"
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
+  def test_string_utils_sha1_hash_nil_value
+    value = nil
+    assert_equal(Digest::SHA1.hexdigest(value.to_s), StringUtils.sha1_hash(value))
+  end
+
```

Comment:
Given that

```
  def self.sha1_hash(value)
    Digest::SHA1.hexdigest value.to_s
  end
```

, these tests are somewhat pointless, seems to me. You are essentially testing that `value.to_s` does not throw, nothing beyond that, seems to me.